lib/fs/fakefs.go

```diff
@@ -259,6 +262,9 @@ func (fs *fakeFS) Chtimes(name string, _ time.Time, mtime time.Time) error {
 	if entry == nil {
 		return os.ErrNotExist
 	}
+	if fs.timePrecisionSecond {
+		mtime = mtime.Round(time.Second)
```

Comment:
If I were a file system this would be truncate and not round, but I guess as this is the more evil behavior it might be better in fakefs :p